backend/sites/src/app/entities/subdivisions.entity.ts

```diff
@@ -9,7 +15,7 @@ import { ChangeAuditEntity } from './changeAuditEntity';
 @Entity('subdivisions')
 export class Subdivisions extends ChangeAuditEntity {
   @Field()
-  @Column('bigint', { primary: true, name: 'id' })
+  @PrimaryGeneratedColumn('increment')
```

Comment:
is this a new primary column? if not, this might create sync issue if id was not set as auto generated.

Reply:
These are both the existing primary columns for these tables.